ruby/plan.ps1

```diff
@@ -13,7 +13,7 @@ $pkg_deps=@(
     "core/openssl"
     "core/zlib"
 )
-$pkg_build_deps=@("core/visual-cpp-build-tools-2015")
+$pkg_build_deps=@("core/visual-cpp-build-tools-2015", "core/zlib", "core/openssl")
```

Comment:
These are both listed as runtime dependencies in `pkg_deps`, so they do not need to be repeated in `pkg_build_deps`.

Reply:
Sweet yep I'll pull those out.